scripts/utils.js

```diff
@@ -59,13 +59,15 @@ export function fuzzyMatchTarget(partialTargets, includeAllMatching) {
  * @param {object} [options]
  */
 export async function exec(command, args, options) {
+  const isWin32 = process.platform === 'win32'
   return new Promise((resolve, reject) => {
     const process = spawn(command, args, {
       stdio: [
         'ignore', // stdin
         'pipe', // stdout
         'pipe', // stderr
       ],
+      shell: isWin32,
```

Comment:
```suggestion
      shell: process.platform === 'win32',
```

Reply:
'process' is declared here, that's why I did it.